lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -363,6 +363,11 @@ def _prepare_environment(  # noqa: C901
                     else:
                         existing_location = node_runbook.location
 
+                # resolve Latest to specified version
+                node_runbook.marketplace = self._parse_marketplace_image(
```

Comment:
It should be called, only if the marketplace is not empty.

Reply:
yes. Just saw the test failures on this. Both location and marketplace need to be specified